setup.cfg

```diff
@@ -54,7 +54,7 @@ image =
 jpeg2000 =
   glymur>=0.8.18,!=0.9.0,!=0.9.5
 map =
-  matplotlib>=3.2.0
+  matplotlib>=3.4.0
```

Comment:
According to our [Dependancy Support Policy](https://docs.sunpy.org/en/latest/dev_guide/contents/dependencies.html) we can't do this.

Matplotlib 3.4 was released in March 2021 meaning we can't depend on it until March 2023 which means 5.0 not 4.1.

What does the html repr rely on in mpl 3.4?

Reply:
If I remember correctly supylabel was only added in 3.4?

Reply:
Yeah that is the problem and why I originally bumped it to 3.4 

Now I need to figure a work around.

Reply:
Delete the line, sorted